XenAdmin/Dialogs/AttachUsbDialog.cs

```diff
@@ -75,6 +107,30 @@ private void BuildList()
             }
         }
 
+        private void treeUsbList_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            UsbItem item = treeUsbList.SelectedItem as UsbItem;
+            if (item != null)
+            {
+                buttonAttach.Enabled = true;
+            }
+            else
+            {
+                buttonAttach.Enabled = false;
+            }
+        }
+
+        private void buttonAttach_Click(object sender, EventArgs e)
+        {
+            UsbItem item = treeUsbList.SelectedItem as UsbItem;
+            System.Threading.ThreadPool.QueueUserWorkItem((System.Threading.WaitCallback)delegate (object o)

```

Comment:
Although we disable the button for items that are not UsbItem, we should still add the null check here for `item`

Reply:
Agree.